openedx/core/djangoapps/olx_rest_api/urls.py

```diff
@@ -0,0 +1,14 @@
+"""
+Studio URL configuration for openedx-olx-rest-api.
+"""
+from django.conf.urls import include, url
+
+from . import views
+
+urlpatterns = [
+    url(r'^api/olx-export/v1/', include([
+        url(r'xblock/(?P<usage_key_str>[^/]+)/$', views.get_block_olx),
+        # Get a static file from an XBlock that's not part of contentstore/GridFS
+        url(r'xblock/(?P<usage_key_str>[^/]+)/export-file/(?P<path>.+)$', views.get_block_exportfs_file),
```

Comment:
Nit: I have a minor preference against nested resources–e.g. `xblock/{usage_key_str}` for the OLX and `file/{usage_key}/<path>`. Mostly because our usage key parsing is a bit wacky.

Not strongly attached to this however, and I'm fine with the approach you've taken.

Reply:
Fixed in 5eb608282f9193df2c1f9395d39113e62ec957e7